frontend/src/components/pages/AdminDashboard/AdminDashboard.tsx

```diff
@@ -12,11 +12,11 @@ import MUIDataTable, {
   CustomHeadLabelRenderOptions,
   MUIDataTableColumn,
 } from "mui-datatables";
-import React from "react";
+import React, { useEffect, useState } from "react";
 
-import { ReviewResponse } from "../../../APIClients/ReviewAPIClient";
+import getReviews from "../../../APIClients/ReviewAPIClient";
+import { ReviewResponse } from "../../../types/ReviewTypes";
 import Author from "./Author";
-import data from "./mockData";
 
 type ReviewRow = {
```

Comment:
should this type be in the ReviewTypes file? I am fine with keeping it here but I think it might be a bit cleaner keeping it in the types file?

Reply:
Hmm I think it might make more sense that this type stays in this file instead of ReviewTypes since it's only being used for the mui datatable row.